tasks/inventory_report/schemas/output.json

```diff
@@ -3,130 +3,114 @@
   "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/inventory_report/schemas/output.json",
   "title": "inventory_report Lambda Output",
   "description": "The output for the inventory_report Lambda.",
-  "type": "array",
-  "items": {
-    "type": "object",
-    "required": [
-      "id",
-      "collections"
-    ],
-    "properties": {
-      "id": {
-        "description": "The unique ID of the provider. Provided by Cumulus.",
-        "type": "string"
-      },
-      "collections": {
-        "description": "Collections within the provider.",
+  "type": "object",
+  "required": [
+    "anotherPage",
+    "granules"
+  ],
+  "properties": {
+    "anotherPage": {

```

Comment:
I can't remember, were we going to pass back a page index value as well for the current page along with this boolean value?